org.eclipse.jdt.ui.tests.refactoring/test cases/org/eclipse/jdt/ui/tests/refactoring/nls/NLSHintStripQuotesTest.java

```diff
@@ -37,76 +37,90 @@ public class NLSHintStripQuotesTest {
 	@Rule
 	public Java15ProjectTestSetup pts15= new Java15ProjectTestSetup();
 
-    private IJavaProject javaProject;
-    private IJavaProject javaProject15;
+	private IJavaProject javaProject;
```

Comment:
please avoid touching so many line with whitespace changes as it disturbs the git history

Reply:
I have restored the old (wrong) whitespaces.